tools/cldr-code/src/main/resources/org/unicode/cldr/draft/keyboard/osx/osx-locales.csv

```diff
@@ -82,7 +82,7 @@ Macedonian,mk,
 Malayalam,ml,
 Malayalam-QWERTY,ml,qwerty
 Maltese,mt,
-Maori,mi,
+Māori,mi,
```

Comment:
Retract, probably want to leave the keyboard ones alone, since they are reflecting what the source data had.